.github/workflows/build.yml

```diff
@@ -7,18 +7,31 @@ on:
 
 jobs:
   run-script:
-    runs-on: ubuntu-latest
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        node: [14]
     env:
       PLATFORM_USER_NAME: ${{ secrets.PLATFORM_USER_NAME }}
       PLATFORM_USER_PASSWORD: ${{ secrets.PLATFORM_USER_PASSWORD }}
+      COVEO_DISABLE_AUTOUPDATE: true
```

Comment:
What is this for ? I searched in the current codebase and could not find anything.

Reply:
It's a hidden flag in `oclif/plugin-update`: